requirements.txt

```diff
@@ -1,9 +1,9 @@
+talisker[gunicorn, prometheus, flask, raven]==0.14.3
 canonicalwebteam.http==1.0.1
+canonicalwebteam.flask_base==0.2.0
```

Comment:
I assume this will become `0.3.1` and you will remove `webapp/app.py` lines 60-78.